aten/src/ATen/cuda/CUDAHalf.cu

```diff
@@ -8,35 +8,35 @@
 namespace at {
 #if CUDA_VERSION < 9000 && !defined(__HIP_PLATFORM_HCC__)
 template <> AT_CUDA_API
-half convert(Half aten_half) {
+half convert<half, Half>(Half aten_half) {
```

Comment:
why do you need all these extra specifications now?

Reply:
At the time, it seemed that nvcc did not understand how to infer the template arguments of the specialization, and I had to explicitly specify them; the two spellings are semantically equivalent. However, that still ended up not being enough, and so the latest version of the patch #11420 handles the problem by having a class which gets specialized.